GTG/gtk/preferences.py

```diff
@@ -210,11 +188,11 @@ class ShortcutWidget(object):
 
     def __init__(self, builder):
         self.builder = builder
-        self.dialog = builder.get_object("PreferencesDialog")
+        self.window = builder.get_object("MainWindow")
         self.button = builder.get_object("shortcut_button")
-        hbox1 = builder.get_object("hbox1")
+        shortcut_box = builder.get_object("shortcut_box")
         self.new_task_default_binding = "<Primary>F12"
-        self.gsettings_install_label_shown = False
+        self.gsettings_install_label_shown = True
```

Comment:
Do you mind to reimplement `GTG.tools.shortcut` with `Gio.Settings`? Have a look at this [AskUbuntu answer](http://askubuntu.com/a/85191/4471) that shows you how you can work with it. ([Unofficial API documentation](http://lazka.github.io/pgi-docs/Gio-2.0/classes/Settings.html))

That should result in much simpler code (no need for long commands) and much faster code (no to do subprocess call).

(I think we can drop support for gconf as the modern distros have migrated off from it -- we can reimplement it back with libraries like python-gconf later if needed)